web/src/main/java/eu/execom/labs/pomodorospringreact/util/GlobalControllerExceptionHandler.java

```
@@ -0,0 +1,32 @@
+package eu.execom.labs.pomodorospringreact.util;
+
+import eu.execom.labs.pomodorospringreact.exceptions.ServiceException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+@ControllerAdvice
+public class GlobalControllerExceptionHandler {
+
+    @ExceptionHandler({ServiceException.class})
+    @ResponseBody
+    public ResponseEntity<?> exceptionHandler(HttpServletRequest request, ServiceException exception) {
+        return createResponseEntity(request, exception, HttpStatus.NOT_FOUND);
```

You can have different version of exceptions which can extend from ServiceException. I would suggest to do the following:
- in DAOs, instead of returning null, throw `EntityNotFoundException`, which extends from RuntimeException
- in Services -> remove null checks and throwing specific exceptions, since if an entity is not found will be covered with an exception being thrown from a DAO
- In this exception handler instead of handling a very generic `ServiceException`, add a handler for specific `EntityNotFoundException`, which will be thrown only in case if an entity is not found.

ServiceException can be used as a parent exception for different kinds of errors, not necessary resulting in a NOT_FOUND error being returned.

This will lead to much cleaner and easier to maintain code.